cpp/order/Steinhardt.h

```diff
@@ -182,9 +174,8 @@ class Steinhardt
     std::shared_ptr<float> m_QliAve;           //!< Averaged Ql with 2nd neighbor shell for each particle i
     std::shared_ptr<complex<float>> m_QlmiAve; //!< Averaged Qlm with 2nd neighbor shell for each particle i
     std::shared_ptr<std::complex<float>> m_QlmAve;   //!< Normalized QlmiAve for the whole system
-    float m_Norm;                                    //!< System normalized norm over all Qlm(Ave)
-    std::complex<float> m_NormWl;                    //!< System normalized norm over all Wlm(Ave)
-    std::shared_ptr<std::complex<float>> m_WliOrder; //!< Wl order parameter for each particle i
+    float m_norm;                                    //!< System normalized order parameter
+    std::shared_ptr<float> m_Wli; //!< Wl order parameter for each particle i
```

Comment:
Can you add a comment thatWli is reused for average as well?

Reply:
Done.